packages/karma/package.json

```diff
@@ -22,19 +22,11 @@
     "yarn": ">=1.2.1"
   },
   "dependencies": {
+    "@neutrinojs/compile-loader": "^8.2.0",
     "@neutrinojs/loader-merge": "^8.2.0",
     "babel-plugin-istanbul": "^4.1.6",
     "deepmerge": "^1.5.2",
-    "karma": "^2.0.2",
-    "karma-chrome-launcher": "^2.2.0",
-    "karma-coverage": "^1.1.2",
-    "karma-mocha": "^1.3.0",
-    "karma-mocha-reporter": "^2.2.5",
-    "karma-webpack": "^3.0.0",
```

Comment:
I think some of these are still needed?

Reply:
I have moved them to peer deps, although I'm not entirely happy about that.